osg-comanage-project-usermap.py

```diff
@@ -146,6 +149,35 @@ def get_osguser_groups(filter_group_name=None):
              for pid, gids in pid_gids.items() }
 
 
+def parse_inputfile(inputfile):
+    user_groupmap = dict()
+    with open(inputfile) as file:
+        for line in file:
+            split_line = line.split()
+            if split_line[0] == "*" and len(split_line) >= 3:
+                user_groupmap[split_line[1]] = split_line[2].split(",")
+    return user_groupmap
+
+
+def merge_maps(maps):
+    while len(maps) > 1:
+        merge_target = maps[0]
+        merge_material = maps.pop(1)
+        for key in merge_material.keys():
+            if key in merge_target:
+                merge_target[key].extend(merge_material[key])
+                merge_target[key] = list(set(merge_target[key]))
+            else:
+                merge_target[key] = merge_material[key]
+    return maps[0]
+
+
+def merge_inputfiles(osguser_groups):
+    maps = [osguser_groups]
+    for inputfile in options.inputfiles:
```

Comment:
```suggestion
    for inputfile in options.inputfiles:
```

Let's avoid implicitly required inputs. I'd make the list of input files an explicitly required function arg.

That being said, why create a function specific to merging input files? To me, the following flow makes sense:

1. Create mapping dict for COmanage groups
2. Create mapping dicts for input files
3. Merge all mapping groups

Which looks like 2 new functions to me for 2 and 3